bin/main

```diff
@@ -24,113 +19,24 @@ def display_board(board)
   puts " #{board[6]} | #{board[7]} | #{board[8]} "
 end
 
-# USER INPUT TO THE ARRAY POSITION
-def input_to_index(user_input)
-  user_input.to_i - 1
-end
-
-# USER MOVEMENT - X or O to a certain position
-def move(board, index, player)
-  board[index] = player
-end
-
-# THE POSITION IS ALREADY TAKEN
-def position_taken?(board, index)
-  if ((board[index]).is_a? Integer) || board[index].nil?
-    false
-  else
-    true
-  end
-end
+# game_on = true
```

Comment:
- [ ] Please delete commented out code and add the .rb extension to the main file. 
Make sure your code is well organized before submiting for a review.